WORKSPACE

```diff
@@ -35,6 +35,7 @@ node_repositories(package_json = [
     "//:package.json",
     "//examples/rollup:package.json",
     "@program_example//:package.json",
+    "//internal/e2e/rollup:package.json",
```

Comment:
opt: rather than actually do the install, you could just have the `node_modules` directory checked in